### PR TITLE
React library

### PR DESCRIPTION
**Pull request by alexlouden**

### CODE REVIEW

File: react/bin/build.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    import PropTypes from 'prop-types';
+
+    const ${ComponentName} = forwardRef(({ color = 'currentColor', size = 21, ...rest }, ref) => {
+      return (
+        <svg ref={ref} ${attrsToString(defaultAttrs)}>
+          ${iconContents}
+        </svg>
+      )
+    });
+    ${ComponentName}.propTypes = {
```
[the remark this answers is not in the document]
Comment (alexlouden) [reply react/bin/build.js:109]: Oh yep good idea - maybe babel-plugin-transform-react-remove-prop-types?

File: react/src/icons/alarm_clock.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React, { forwardRef } from 'react'
```
[the remark this answers is not in the document]
Comment (alexlouden) [reply react/src/icons/alarm_clock.js:1]: Maybe nice to check them in just so people can read the source, see diffs, copy paste out individual icons, etc? I find that really handy when using react-feather

[the remark this answers is not in the document]
Comment (JakeGinnivan) [reply react/src/icons/alarm_clock.js:1]: That's fair. Maybe add compile to prepare or prepublish so it's not possible to forget to update?

File: react/bin/build.js [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    fill: 'none',
+    stroke: 'color',
+    strokeWidth: 1,
+    strokeLinecap: 'round',
+    strokeLinejoin: 'round',
+    rest: '...rest',
+  }
+
+  const element = `
+    import React, {forwardRef} from 'react';
```
[the remark this answers is not in the document]
Comment (alexlouden) [reply react/bin/build.js:99]: Or could use rollup for `main` like react-feather does? I've just been following the same structure, and wasn't sure that was necessary

https://github.com/feathericons/react-feather/blob/master/package.json#L5-L13

File: react/src/icons/alarm_clock.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexlouden) [reply react/src/icons/alarm_clock.js:1]: Yeah good idea, would be nice to get these into storybook at some point too - would be nice to be able to test them locally before publishing

[the remark this answers is not in the document]
Comment (alexlouden) [reply react/src/icons/alarm_clock.js:1]: I left it as a manual step just so you have a bit more control over the process, and you can check the output manually etc when editing the build script - but definitely room to automate in the future

**Pull request closed without merging**